doc/api/module.md

```diff
@@ -217,6 +217,48 @@ added: v22.8.0
 * Returns: {string|undefined} Path to the [module compile cache][] directory if it is enabled,
   or `undefined` otherwise.
 
+### `module.getPackageJSON(startLocation[, everything])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active Development
+
+* `startLocation` {string} A fully resolved FS path or file URL to start looking
+* `everything` {boolean} Whether to return the full contents of the found package.json
+* Returns: {Object | undefined}
```

Comment:
```suggestion
* Returns: {Object|undefined}
```
Ditto to the rest

Reply:
why?

Reply:
It's the way the docs are formatted. There is no space between the `|`.

Reply:
Does it actually matter or is it a style/consistency issue?

Reply:
I don't think the linter will throw an error, but it's documented in the Style Guide (https://github.com/nodejs/node/blob/main/doc/README.md#function-arguments-and-returns)

Reply:
Yeah let's keep them consistent, i.e. without spaces